infra/recipes/terraform/azure/modules/vm/outputs.tf

```diff
@@ -0,0 +1,7 @@
+output "virtual_network_name" {
+    value = azurerm_virtual_network.name
+}
+
+output "vm_size" {
+    value = azurerm_virtual_machine.vm.vm_size
+}
```

Comment:
I am wondering why did you select only these two variables as output variables? 

Output variables are used for logging during terraform apply and inter-module communication.

Reply:
I selected these two because the mysql module needs the virtual network name and the k8s module needs the vm size.